BassBotDotNET/Program.cs

```diff
@@ -4,14 +4,23 @@
 using Discord.WebSocket;
 using Newtonsoft.Json;
 
+namespace BassBotDotNET;
+
+/// <summary>
+/// Main program to run the discord bot.
+/// </summary>
 public class Program
 {
-    public static Task Main(string[] args) => new Program().MainAsync();
+    public static Task Main() => new Program().MainAsync();
 
     private DiscordSocketClient? _client;
 
+    // Settings command module
     private readonly SettingSlashCommandModule _settingSlashCommandModule = new();
 
+    /// <summary>
+    /// Main method for the bot.
+    /// </summary>
```

Comment:
Ohhh is this similar to the Constructor in Java?

Reply:
Nope that is not a Constructor in C#

Reply:
xD This is more of a the .NET specific library for discord. I'm not entirely sure how it works either though.